vtexdocs/devportal
# Feature/VTEX IO Page Layout

#### What is the purpose of this pull request?

To add the correct layout (as per [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2641%3A149733)) to the VTEX IO page of the Developers Portal.

Documented [here](https://www.notion.so/vtexhandbook/Criar-p-gina-VTEX-IO-com-a-estrutura-do-Figma-339d26aefcea4cd2b880d1cbae1b7f17)

#### What problem is this solving?

Currently, the VTEX IO page of the Developers Portal is empty. It should contain the sidebar and some content next to it. This pull request implements that layout and adds some of the content of the page.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-39--elated-hoover-5c29bf.netlify.app/) and check the layout against [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2641%3A149733) with different breakpoints, keeping in mind the cards in between the "Welcome" and "Other resources" sections will be implemented in future PRs.

#### Screenshots or example usage

#### Types of changes
